app/javascript/Application/components/Print/PrintAssessment.test.js

```
@@ -4,13 +4,29 @@ import PrintAssessment from './PrintAssessment'
 import { assessmentPrint, i18nPrint, printFixture } from '../Assessment/assessment.mocks.test'
 
 describe('<PrintAssessment />', () => {
+  let printAssessment, printAssessmentHtml
+
+  beforeEach(() => {
+    printAssessment = shallow(<PrintAssessment assessment={assessmentPrint} i18n={i18nPrint} />)
+    printAssessmentHtml = printAssessment.html()
+  })
+
   it('should match fixture as plain html', () => {
-    const printAssessment = shallow(<PrintAssessment assessment={assessmentPrint} i18n={i18nPrint} />)
-    const actualHtmlRaw = printAssessment.html()
-    const actualHtml = actualHtmlRaw.replace(
+    const actualHtml = printAssessmentHtml.replace(
       /<span style="text-align:right;font-style:italic">[^>]+<\/span>/g,
       '<span style="text-align:right;font-style:italic" />'
     )
     expect(actualHtml).toEqual(printFixture)
   })
+
+  it('should render confidential item and redact rating when confidential by default and confidential is true', () => {
+    expect(printAssessmentHtml).not.toContain('Confidential')
+    expect(printAssessmentHtml).not.toContain('<div style="width:7.6rem"></div>') // Redacted rating
```

This seems very fragile. If somebody changes the style, for example, this will silently pass even if it is not redacting the ratings.